src/components/AppHelp.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+	<div>
+		<AppSettingsDialog :open.sync="settingsOpen" :showNavigation="true">
+			<div class="exit icon-close" @click="settingsOpen = false; "/>
+			<h2>{{ t('notes', 'Notes Application') }}</h2>
+			<AppSettingsSection :title="t('notes', 'Basics')">
+				<div class="feature icon-add">
+					{{ t('notes', 'Start writing a note by clicking on “{newnote}” in the app navigation.', { newnote: t('notes', 'New note') }) }}
+				</div>
+				<div class="feature icon-fullscreen">
+					{{ t('notes', 'Write down your thoughts without any distractions.') }}
+				</div>
+				<div class="feature icon-files-dark">
+					{{ t('notes', 'Organize your notes in categories.') }}
+				</div>
+				<button class="button-icon-add icon-add" @click="onNewNote">
+					{{ t('notes', 'Create a sample note with markdown') }}
+				</button>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Markdown')">
+				<div class="feature icon-toggle-filelist">
+					{{ t('notes', 'Use Markdown markups to style your text.') }}
+				</div>
+				<button @click="onNewNote">
+					{{ t('notes', 'Create a new Note with markdown samples') }}
+				</button>
+				<table>
+					<tr>
+						<th>
+							{{ t('notes', 'Sequence') }}
+						</th>
+						<th>
+							{{ t('notes', 'Result') }}
+						</th>
+						<th>
+							{{ t('notes', 'Visualized') }}
+						</th>
+					</tr>
+					<tr v-for="item in getMarkdown">
+						<td v-html="item.sequence">
+							{{ item.sequence }}
+						</td>
+						<td>
+							{{ item.result }}
+						</td>
+						<td v-html="item.visualized">
+							{{ item.visualized }}
+						</td>
+					</tr>
+				</table>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Shortcuts')">
+				<div class="feature icon-toggle-filelist">
+					{{ t('notes', 'Use shortcuts to quickly navigate this app.') }}
+				</div>
+				<table>
+					<tr>
+						<th>{{ t('notes', 'Shortcut') }}</th>
+						<th>{{ t('notes', 'Action') }}</th>
+					</tr>
+					<tr v-for="item in getShortcuts">
+						<td>{{ item.shortcut }}</td>
+						<td>{{ item.action }}</td>
+					</tr>
+				</table>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Tips and Tricks')">
+				<div class="feature icon-toggle-filelist">
+					{{ t('notes', 'Double Click the text in viewmode to quickly open edit mode!') }}
+				</div>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Apps')">
+				<div class="feature icon-phone">
+					{{ t('notes', 'Install the app for your mobile phone in order to access your notes from everywhere.') }}
+				</div>
+				<div class="badge-wrapper">
+					<a href="https://github.com/stefan-niedermann/nextcloud-notes">
```

Comment:
I think we should open all external links in a new window/tab.
```suggestion
					<a target="_blank" href="https://github.com/stefan-niedermann/nextcloud-notes">
```